### PR TITLE
Allow default values for environment variables

### PR DESCRIPTION
**Pull request by f1x3d**
Inspired by your [issue on the archived repo](https://github.com/bradleygolden/pytailor/issues/9).

Also, I've removed the `NOT_FOUND` constant used for checking if the value exists in the dictionary, since it might be an actual valid value (see the `test_from_env_var_that_equals_not_found` test for an example).

### CODE REVIEW

**Comment by bradleygolden** [comment]
Thanks, I'll take a look in the next few days.

File: tests/test_core.py [hunk 1]
```diff
@@ -149,3 +149,30 @@ def test_env_var_can_be_renamed():
...
+    assert config["FOO"] == "BAR"
+    del os.environ["FOO"]
+    assert config["FOO"] == "BAZ"
+
+
+def test_from_env_var_with_default_value_and_alias():
+    config = Config()
+    os.environ["FOO"] = "BAR"
+    config.from_envar("FOO", rename="FOOBAR", default="BAZ")
+    assert config["FOO"] == "BAR"
```
[the remark this answers is not in the document]
Comment (bradleygolden) [reply tests/test_core.py]: Confirmed on my end, this is definitely a bug that already existed.

**Comment by bradleygolden** [comment]
First, I want to to say, thanks so much for contribution!  There are a few things that need to be done before I can merge this.

- [ ] Update the `versions.py` with a patch change (0.1.1). This is technically a feature but we will call it a patch for now since this is still a prerelease.
- [ ] Update the `CHANGELOG.md` to reflect the changes you've made.
- [ ] Add yourself as a contributor to `AUTHORS.md`. You deserve it!

A takeaway for me is to add this to the README so that it's more clear to future contributors.

**Comment by f1x3d** [comment]
@bradleygolden Not sure why codecov failed this time, but here you go!

**Pull request merged**